run/events-pubsub/README.md

```diff
@@ -0,0 +1,51 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes Pub/Sub messages.
+
+[![Run in Google Cloud][run_img]][run_link]
+
+[run_img]: https://storage.googleapis.com/cloudrun/button.svg
+[run_link]: https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_image=gcr.io/cloudrun/button&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/ruby-docs-samples&cloudshell_working_dir=run/events-pubsub
+
+## Quickstart
+
+Create a Cloud Pub/Sub topic:
+
+```
+gcloud pubsub topics create my-topic
+```
+
+Create a Cloud Pub/Sub topic:
+
+```
+gcloud alpha events triggers create pubsub-trigger-ruby \
+--target-service run-events-pubsub \
+--type com.google.cloud.pubsub.topic.publish \
+--parameters topic=my-topic
+```
+
+Deploy your Cloud Run service:
+
+```
+gcloud builds submit \
+ --tag gcr.io/$(gcloud config get-value project)/run-events-pubsub
```

Comment:
Looks like other languages are using `cloudrun-events-pubsub` as the container name

Reply:
Sounds good